### DISCUSSION TITLE
Rust project in a sub-folder

### DISCUSSION THREAD

**Discussion by hckhanh** [rust-lang/rust-clippy#9615] [Q&A] [answered]
How can I specify the `Cargo.toml` to run with `clippy` if the Rust project is located in a sub-folder

**Comment by flip1995** [comment, marked as answer]
I'm not sure if I understand the question correctly. Can you give a bit more details about your project structure, please. Is it a workspace, in that case with `cargo clippy -p package_name`. If it is just a crate in a sub folder then `cd` into it and run `cargo clippy` in there.

Or are you talking about Clippy development?

**Reply by hckhanh** [reply]
> I'm not sure if I understand the question correctly. Can you give a bit more details about your project structure, please. Is it a workspace, in that case with `cargo clippy -p package_name`. If it is just a crate in a sub folder then `cd` into it and run `cargo clippy` in there.
> 
> 
> 
> Or are you talking about Clippy development?

Hi, I have a npm project with a rust project inside. So I guest that it's a sub folder instead of a workspace.
